api/main.go

```diff
@@ -114,6 +114,8 @@ func main() {
 			fireball.LogDecorator(),
 			fireball.BasicAuthDecorator(user, pass))
 
+		routes = append(routes, controllers.NewHealthController(cfg, Version).Routes()...)
```

Comment:
this should probably have a comment above, like `// don't decorate health endpoint with auth`

Reply:
:white_check_mark: